src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -20,7 +22,7 @@ public Integer oddSum(List<Integer> numbers) {
      * Вернуть количество вхождений объекта `element`</p>
      **/
     public Long elementCount(List<String> elements, String element) {
-        return null;
+        return elements.stream().filter(x -> x.equals(element)).count();
```

Comment:
```suggestion
        return elements.stream()
        .filter(x -> x.equals(element))
        .count();
```
write your streams like this for better readability